lib/validate.js

```diff
@@ -748,14 +748,32 @@ export const nwcSchema = object().shape({
   })
 }, ['nwcUrl', 'nwcUrlRecv'])
 
-export const blinkSchema = object({
+export const blinkSchema = object().shape({
   apiKey: string()
-    .required('required')
-    .matches(/^blink_[A-Za-z0-9]+$/, { message: 'must match pattern blink_A-Za-z0-9' }),
+    .matches(/^blink_[A-Za-z0-9]+$/, { message: 'must match pattern blink_A-Za-z0-9' })
+    .when(['apiKeyRecv'], ([apiKeyRecv], schema) => {
+      if (!apiKeyRecv) return schema.required('required if api key for receiving not set')
+      return schema.test({
+        test: apiKey => apiKey !== apiKeyRecv,
+        message: 'api key for sending cannot be the same as for receiving'
+      })
+    }),
+  apiKeyRecv: string()
+    .matches(/^blink_[A-Za-z0-9]+$/, { message: 'must match pattern blink_A-Za-z0-9' })
+    .when(['apiKey'], ([apiKey], schema) => {
+      if (!apiKey) return schema.required('required if api key for sending not set')
+      return schema.test({
+        test: apiKeyRecv => apiKeyRecv !== apiKey,
+        message: 'api key for receiving cannot be the same as for sending'
+      })
+    }),
   currency: string()
     .transform(value => value ? value.toUpperCase() : 'BTC')
-    .oneOf(['USD', 'BTC'], 'must be BTC or USD')
-})
+    .oneOf(['USD', 'BTC'], 'must be BTC or USD'),
+  currencyRecv: string()
+    .transform(value => value ? value.toUpperCase() : 'BTC')
+    .oneOf(['BTC'], 'must be BTC')
+}, ['apiKey', 'apiKeyRecv'])
```

Comment:
You need to add `...autowithdrawSchemaMembers` here for validation of autowithdraw settings to work, see `LNDAutowithdrawSchema` for example. I did that mistake in NWC, LNbits and phoenixd, see #1487,